Chap_API_Struct.tex

```diff
@@ -3586,8 +3586,8 @@ \subsection{Identification attributes}
 }
 
 %
-\declareAttributeNEW{PMIX_PSET_NAME}{"pmix.pset.nm"}{char*}{
-User-assigned name for the process set containing the given process.
+\declareAttributeNEW{PMIX_PSET_NAMES}{"pmix.pset.nm"}{char*}{
+Comma-delimited list of process sets and process groups to which the given process is a member.
```

Comment:
Should we split this into `PMIX_PSET_NAMES` and `PMIX_PGROUP_NAMES`? For the same reason as the query attributes.

Reply:
Yeah, and I moved them to the process set/group chapter with the rest of those related attributes.